apps/cloud/odc/apps/cloud/s3_find.py

```diff
@@ -1,13 +1,17 @@
 import click
 import sys
-from odc.aio import s3_find_glob
+from odc.aio import s3_find_glob, S3Fetcher
 
 
 @click.command('s3-find')
 @click.option('--skip-check', is_flag=True,
               help='Assume file exists when listing exact file rather than wildcard.')
+@click.option('--no-sign-request', is_flag=True,
+              help='Do not sign AWS S3 requests')
+@click.option('--request-payer', is_flag=True,
```

Comment:
The AWS CLI uses `--request-payer requester` as the flag, which sucks. This way is better. Just saying ;-)

Reply:
I guess that makes sense, oh well let's just leave it as is, unless you can suggest a better name to not be "almost like aws but not really". Maybe `--requestor-pays`? But then it's too similar but different.

Reply:
I guess with the current option name you can turn it into sentence as "this command is `--request-payer`"